Cargo.toml

```diff
@@ -17,9 +17,12 @@ cap-std = "0.24.1"
 oci-spec = "0.5"
 thiserror = "1.0"
 serde_json = "1.0"
+nix = "0.25"
 
 [dev-dependencies]
 tempfile = "3.0"
+libc = "0.2.135"
```

Comment:
It looks like most of our dependencies are using `major.minor`. How about we use `0.2` for `libc`, and also get rid of the patch versioning for `cap-std`, and `chorno`?